polkadot/primitives/src/vstaging/mod.rs

```diff
@@ -66,8 +68,9 @@ pub enum CandidateDescriptorVersion {
 }
 
 /// A unique descriptor of the candidate receipt.
-#[derive(PartialEq, Eq, Clone, Encode, Decode, DecodeWithMemTracking, TypeInfo, RuntimeDebug)]
+#[derive(PartialEq, Eq, Clone, Encode, Decode, DecodeWithMemTracking, TypeInfo)]
 #[cfg_attr(feature = "std", derive(Hash))]
+#[cfg_attr(not(feature = "std"), derive(RuntimeDebug))]
```

Comment:
```suggestion
```

Reply:
I was thinking we want to keep the empty implementation for when std is not enabled, similarly to what it was before this PR.

Reply:
I'm slowly removing this everywhere, it just makes stuff more complicated when you want to debug stuff.